### ISSUE TITLE
Triple-store deliverables (hackathon)

### ISSUE THREAD

**Issue by tmillross** [BONSAMURAIS/triple-store#1]
Tasks are grouped under headers for 1. widget-related, 2. non-widget-related, 3. links to stretch goals in separate issues.

### Access/credentials
- [x] ~~Get Aalborg instance server credentials~~ (_not used, see #3_)
- [x] Get admin credentials on server (Tom & Tomas have)
- [ ] Create non-admin users (as required for apps/interfaces, see #5 )
_Fuseki access has been a struggle at the hackathon_
- [x] Consider appropriate credential management within Bonsai organisation (out of repo scope)

Data Loading
- [x] EXIOBASE data imported 
- [x] Electricity model ([Bentso](https://github.com/BONSAMURAIS/bentso)) data imported

Download and export page
 - [x] Test on dummy data (done: https://db.bonsai.uno/foaf/data )
 - [x] Export specification (RDF as JSON-LD)
 - [x] General export function (automatically created through Fuseki interface). 
Specific export function, using queries on named graphs containing for example: 
 - [ ] Raw database
 - [ ] Resolved database
_Maybe accessed/called via the [web-app](https://app.bonsai.uno/) ?_

## Non-widget hackathon deliverables

SPARQL endpoint
 - [x] Test with dummy data https://db.bonsai.uno/foaf/query
 - [x] publicly accessible (_currently pw protected whilst in dev_)

Documentation
- [x] Functional readme following BEP0004 suggestions
- [x] Write instructions: configuring a new db instance. Such that someone new can set up their own DB instance (reproducibility)

## Stretch goals
_Stored in separate issues to not impact the progress bars_
See #4

Data versioning
Raw data stored in a separate graph from resolved data 
_conceptual work needed for the provenance of derived data? _

**Comment by mmr2187** [comment]
I think we're waiting for the final version of the ontology to be able to do the imports right? or do we just do the mapping afterwards?

**Comment by tmillross** [comment]
Yup. I think the dependencies are...

1.  ontology draft ready
2.  data (instances) following that ontology

Then when that data needs to be stored: we should ideally be ready to receive it immediately. Presumably using the existing Jena instance at Aalborg. Do you (or anyone?) know the details about that? IP, credentials etc.?

**Comment by mmr2187** [comment]
nope, no details for that from me. perhaps ask in #general in slack.

**Comment by kuzeko** [comment]
The ontology should be ready to be loaded. 
Please import it in named graph `<http://ontology.bonsai.uno/core>`

Also, there should be somewhere the named graph for the output of the correspondance table data I guess
